packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx

```diff
@@ -141,9 +141,10 @@ export const LiveblocksPlugin = ({
 
   const providerFactory = useCallback(
```

Comment:
@nvie this is the provider factory that's given to Lexical, which requires a new provider each time it is executed.